model/app/registry.yaml

```diff
@@ -31,3 +31,34 @@ groups:
           More information about Android identifier best practices can be found [here](https://developer.android.com/training/articles/user-data-ids).
         examples:
           - 2ab2916d-a51f-4ac8-80ee-45ac31a28092
+      - id: app.screen.coordinate.x
+        type: int
+        brief: The x (horizontal) coordinate of a screen coordinate, in screen pixels.
+        stability: development
+        examples: [ 0, 131 ]
+      - id: app.screen.coordinate.y
```

Comment:
@breedx-splk @MSNev do you think we can be consistent (one way or the other) with how we represent coordinates across client instrumentation?

related to https://github.com/open-telemetry/semantic-conventions/pull/1941#discussion_r2001519799

Reply:
> @breedx-splk @MSNev do you think we can be consistent (one way or the other) with how we represent coordinates across client instrumentation?
> 
> related to [#1941 (comment)](https://github.com/open-telemetry/semantic-conventions/pull/1941#discussion_r2001519799)

Seems unlikely, especially if browser is pushing for a 2-element array. That's way too magical for me.

Additionally, having separate fields allows filtering/processing/whatever without parsing/splitting/extracting.

Reply:
We also don't have this as a top level attribute as it will be part of the event content and not directly comparable across different events.

nit: The `type` says `int`, but you example is an array?

Reply:
> nit: The type says int, but you example is an array?

That's a yaml array, containing more than one integer example. Would it help if I expanded those to a multi-line list format?